iis-wf/iis-wf-ingest-pmc/src/test/java/eu/dnetlib/iis/wf/ingest/pmc/metadata/PmcXmlHandlerTest.java

```diff
@@ -33,10 +33,10 @@
  */
 public class PmcXmlHandlerTest {
 
-	Reader fileReader;
-	SAXParser saxParser;
-	ExtractedDocumentMetadata.Builder metaBuilder;
-	PmcXmlHandler pmcXmlHandler;
+	private Reader fileReader;
+	private SAXParser saxParser;
+	private ExtractedDocumentMetadata.Builder metaBuilder;
```

Comment:
You don't test contr-aff relations in the method testParsingAffiliations. And it would be great to test them here, because different mixed aff types are in the xml file.